### PR TITLE
Move course duplication functionality into its own page rather than a modal

### LINKED ISSUE #556: Remove course duplication from modal
Can non-LTI course duplication be a regular screen instead of a modal? This would be more consistent with other workflows.

### PR DESCRIPTION
**Pull request by wynnset**
Closes #556

### CODE REVIEW

**Comment by lenglund** [comment]
Can you add spacing between the "Back" and "Duplicate" buttons to match what appears between the "Save Draft" and "Submit" buttons?

**Comment by lenglund** [comment]
Also, if the Back button means I will lose changes (e.g., if I manually change dates for the assignments, these will revert when I leave and come back), I'd suggest we also add a confirmation dialogue letting the user know s/he will lose changes (but only IF changes have been made): "Are you sure you want to leave this page? Any dates you've manually changed for assignments here will be lost."

**Comment by andrew-gardener** [comment]
@wynnset I'll try and get you setup for testing LTI links when you're back in

**Comment by wynnset** [comment]
@andrew-gardener sounds good thanks.

**Comment by wynnset** [comment]
@lenglund / @andrew-gardener please re-review. I think this should be ready to get merged.

File: compair/static/modules/course/course-module.js [hunk 1]
```diff
@@ -463,11 +454,9 @@ module.controller(
                 Session.expirePermissions();
 
                 var course = ret;
-                if ($scope.closeDuplicate) {
-                    $scope.closeDuplicate(course.id);
-                } else {
-                    $scope.$close(course.id);
-                }
+                submitted = false;
+                $location.path('/course/' + ret.id);
```
Comment (andrew-gardener) [inline compair/static/modules/course/course-module.js]: Sorry, I missed this the first time but this breaks LTI links. the course id needs to be passed back to the LTI controller via modal closing in order for the new duplicated course to be linked to the lti context

Comment (wynnset) [reply compair/static/modules/course/course-module.js]: Ok, thanks. Added that in now.

**Comment by lenglund** [comment]
Please avoid inline styles and add to the stylesheets instead. Thx!

**Pull request merged**